src/server/game/Entities/Player/Player.cpp

```diff
@@ -18672,31 +18672,31 @@ bool Player::Satisfy(AccessRequirement const* ar, uint32 target_map, bool report
 
 bool Player::CheckInstanceLoginValid(Map* map)
 {
-    if (!map->IsDungeon() || IsGameMaster())
+    if (!map->IsDungeon() || IsInstanceLoginGameMasterException())
```

Comment:
Isn't this going to send the ingame message even if the player is actually in a raid/group ?

Please provide a list of the tests you did ingame.